sdk/src/store.rs

```diff
@@ -4055,10 +4072,19 @@ impl Store {
                     crate::utils::xmp_inmemory_utils::XmpInfo::from_source(stream, asset_type)
                         .provenance
                 {
-                    Ok((
-                        Store::handle_remote_manifest_async(&ext_ref).await?,
-                        Some(ext_ref),
-                    ))
+                    #[cfg(not(target_arch = "wasm32"))]
+                    return Ok((Store::handle_remote_manifest(&ext_ref)?, Some(ext_ref)));
+                    #[cfg(target_arch = "wasm32")]
```

Comment:
Here too, I think we may be able to naturally support async remote manifest calls for non-wasm32 targets.

Reply:
Yes. We can discuss and raise a second PR if we want a non-wasm async version.